core/src/helpers.rs

```diff
@@ -226,7 +226,7 @@ pub fn starknet_block_to_eth_block(block: MaybePendingStarknetBlock) -> RichBloc
                     pending_block_with_txs
                         .transactions
                         .into_iter()
-                        .map(starknet_tx_into_eth_tx)
+                        .map(|t| starknet_tx_into_eth_tx(t, FieldElement::ZERO, 0))
```

Comment:
Let's change this to `None` in case it's not available

Reply:
Done